tex/CT4P.tex

```diff
@@ -495,7 +495,7 @@ \section{Categories}
 The proof that left unit law and associator law hold are analogous.
 \end{exa}
 
-\begin{exer}
+\begin{exer}\label{exer:post_antisymmetry}
```

Comment:
Here and elsewhere, I suggest we add a pointer to the solution, like this (or in a better format):
```suggestion
\begin{exer}[See \cref{label-of-solution}]\label{exer:post_antisymmetry}
```
It is best to use the same label as the exercise, but with the prefix "sol:" or "solution:".